test(integration): Added custom image pull secrets test (#3808)
SumoLogic/sumologic-kubernetes-collection: deploy/helm/sumologic/README.md

| `kube-prometheus-stack.kube-state-metrics.customLabels`                               | Custom labels to apply to service, deployment and pods. Uses sub-chart defaults.                                                                                                                                                                                                                           | `{}`                                                                                                                                                                                 |
| `kube-prometheus-stack.kube-state-metrics.podAnnotations`                             | Additional annotations for pods in the DaemonSet. Uses sub-chart defaults.                                                                                                                                                                                                                                 | `{}`                                                                                                                                                                                 |

> | `kube-prometheus-stack.kube-state-metrics.imagePullSecrets`                           | Pull secrets for falco images. For more information on using Kubernetes secrets with container registries please refer to [Creating a Secret with a Docker config at kubernetes.io](https://kubernetes.io/docs/concepts/containers/images/                                                                 | `[]`                                                                                                                                                                                 |

Reviewer: ```suggestion
| `kube-prometheus-stack.kube-state-metrics.imagePullSecrets`                           | Pull secrets for Kube Prometheus Stack images. For more information on using Kubernetes secrets with container registries please refer to [Creating a Secret with a Docker config at kubernetes.io](https://kubernetes.io/docs/concepts/containers/images/                                                                 | `[]`                                                                                                                                                                                 |
```

| `kube-prometheus-stack.prometheus-node-exporter.podLabels`                            | Additional labels for prometheus-node-exporter pods.                                                                                                                                                                                                                                                       | `{}`                                                                                                                                                                                 |
| `kube-prometheus-stack.prometheus-node-exporter.podAnnotations`                       | Additional annotations for prometheus-node-exporter pods.                                                                                                                                                                                                                                                  | `{}`                                                                                                                                                                                 |

> | `kube-prometheus-stack.prometheus-node-exporter.imagePullSecrets`                     | Pull secrets for falco images. For more information on using Kubernetes secrets with container registries please refer to [Creating a Secret with a Docker config at kubernetes.io](https://kubernetes.io/docs/concepts/containers/images/                                                                 | `[]`                                                                                                                                                                                 |

Reviewer: ```suggestion
| `kube-prometheus-stack.prometheus-node-exporter.imagePullSecrets`                     | Pull secrets for Prometheus Node Exporter images. For more information on using Kubernetes secrets with container registries please refer to [Creating a Secret with a Docker config at kubernetes.io](https://kubernetes.io/docs/concepts/containers/images/                                                                 | `[]`                                                                                                                                                                                 |
```

Author: ahhhh will fix

| `tracesGateway.autoscaling.targetCPUUtilizationPercentage`                            | The desired target CPU utilization for autoscaling.                                                                                                                                                                                                                                                        | `100`                                                                                                                                                                                |
| `tracesGateway.autoscaling.targetMemoryUtilizationPercentage`                         | The desired target memory utilization for autoscaling.                                                                                                                                                                                                                                                     | `50`                                                                                                                                                                                 |

> | `tracesGateway.deployment.imagePullSecrets`                                           | Pull secrets for falco images. For more information on using Kubernetes secrets with container registries please refer to [Creating a Secret with a Docker config at kubernetes.io](https://kubernetes.io/docs/concepts/containers/images/                                                                 | `[]`                                                                                                                                                                                 |

Reviewer: ```suggestion
| `tracesGateway.deployment.imagePullSecrets`                                           | Pull secrets for OpenTelemetry Collector images. For more information on using Kubernetes secrets with container registries please refer to [Creating a Secret with a Docker config at kubernetes.io](https://kubernetes.io/docs/concepts/containers/images/                                                                 | `[]`                                                                                                                                                                                 |
```

Reviewer: Is it used somewhere?

Author: commented
